### ISSUE TITLE
ScribeFire next 4.2.4 does not allow entereing new Blogs one after another

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#783]
```
What's the problem?
I wanted to enter a series of Blogs. by clicking on 'Neues Blog hinzufügen'. 
Enter first Blog - worked fine. Clicking again on 'Neues Blog hinzufügen' - 
the Window with the slider allowing to enter Blog-Adress came up, but I could 
NOT enter the Blogs URL

What browser are you using?
Firefox 25.0

What version of ScribeFire are you running?
SribeFire next 4.2.4

```

-----
Original issue reported on code.google.com by `chri...@gmail.com` on 6 Nov 2013 at 8:32

**Comment by GoogleCodeExporter** [comment]
```
please update to 4.3.1 and let us know if that resolves your issue
```

-----
Original comment by `scribefi...@gmail.com` on 10 Jun 2014 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
this is fixed in version 4.3.3
```

-----
Original comment by `scribefi...@gmail.com` on 18 Jun 2014 at 1:38
* Changed state: **Fixed**
